Library/ViewModels/ProjectPamphletMainCellViewModelTests.swift

```diff
@@ -623,44 +629,16 @@ final class ProjectPamphletMainCellViewModelTests: TestCase {
       XCTAssertEqual(self.optimizelyClient.trackedAttributes?["session_device_type"] as? String, "phone")
     }
   }
+}
 
-  func testTrackingProjectViewed_LiveProject_LoggedIn_NonBacked() {
-    let user = User.template
-      |> \.location .~ Location.template
-      |> \.stats.backedProjectsCount .~ 50
-
-    let project = Project.template
-      |> Project.lens.state .~ .live
-      |> Project.lens.personalization.backing .~ nil
-
-    withEnvironment(currentUser: user) {
-      self.vm.inputs.configureWith(value: (project, .discovery))
-      self.vm.inputs.awakeFromNib()
-
-      XCTAssertEqual(self.dataLakeTrackingClient.events, ["Page Viewed"])
-      XCTAssertEqual(self.segmentTrackingClient.events, ["Page Viewed"])
-
-      XCTAssertEqual(self.dataLakeTrackingClient.properties(forKey: "context_page"), ["project"])
-      XCTAssertEqual(self.dataLakeTrackingClient.properties(forKey: "context_section"), ["campaign"])
-      XCTAssertEqual(self.dataLakeTrackingClient.properties(forKey: "session_ref_tag"), ["discovery"])
-      XCTAssertEqual(self.segmentTrackingClient.properties(forKey: "context_page"), ["project"])
-      XCTAssertEqual(self.segmentTrackingClient.properties(forKey: "context_section"), ["campaign"])
-      XCTAssertEqual(self.segmentTrackingClient.properties(forKey: "session_ref_tag"), ["discovery"])
-
-      XCTAssertEqual(self.dataLakeTrackingClient.properties(forKey: "project_subcategory"), ["Art"])
-      XCTAssertEqual(self.dataLakeTrackingClient.properties(forKey: "project_category"), [nil])
-      XCTAssertEqual(self.dataLakeTrackingClient.properties(forKey: "project_country"), ["US"])
-      XCTAssertEqual(
-        self.dataLakeTrackingClient.properties(forKey: "project_user_has_watched", as: Bool.self),
-        [nil]
-      )
-      XCTAssertEqual(self.segmentTrackingClient.properties(forKey: "project_subcategory"), ["Art"])
-      XCTAssertEqual(self.segmentTrackingClient.properties(forKey: "project_category"), [nil])
-      XCTAssertEqual(self.segmentTrackingClient.properties(forKey: "project_country"), ["US"])
-      XCTAssertEqual(
-        self.segmentTrackingClient.properties(forKey: "project_user_has_watched", as: Bool.self),
-        [nil]
-      )
-    }
+private func assertProjectRefTagEqual(
```

Comment:
Can this be replaced with a standard assertion like we use everywhere else?

Reply:
`XCTAssertEqual` doesn't work on tuples because they don't conform to `Equatable` protocol that's why I had to do that. Another option would be to pass the `Project` and `RefTag` as separate parameters to the `config()` method of `ProjectDescriptionViewController`